src/express/common/config-utils.js

```diff
@@ -480,10 +595,55 @@ export async function editMaticCliRemoteYAMLConfig() {
   )
 
   setCommonConfigs(doc)
-  setConfigList('devnetBorHosts', process.env.DEVNET_BOR_HOSTS, doc)
-  setConfigList('devnetHeimdallHosts', process.env.DEVNET_BOR_HOSTS, doc)
-  setConfigList('devnetBorUsers', process.env.DEVNET_BOR_USERS, doc)
-  setConfigList('devnetHeimdallUsers', process.env.DEVNET_BOR_USERS, doc)
+  if (!process.env.DEVNET_BOR_USERS) {
+    setConfigList('devnetErigonHosts', process.env.DEVNET_ERIGON_HOSTS, doc)
+    setConfigList('devnetErigonUsers', process.env.DEVNET_ERIGON_USERS, doc)
+    setConfigList('devnetHeimdallUsers', process.env.DEVNET_ERIGON_USERS, doc)
+    setConfigList('devnetHeimdallHosts', process.env.DEVNET_ERIGON_HOSTS, doc)
+    deleteConfig('devnetBorUsers', doc)
```

Comment:
Why do we set earlier if we know we'll delete afterwards?
This whole process was meant to have bor only, but maybe now it's about time to refactor it in a way it's suitable for more clients, without too many complications. Maybe we can discuss new approaches over a call